src/modules/dracut/main.py

```diff
@@ -35,15 +35,22 @@ def run_dracut():
 
     :return:
     """
+    # Fetch the job configuration
+    cli_options = ["-f"]
+    initramfs_name = libcalamares.job.configuration.get('initramfsName', None)
+    dracut_options = libcalamares.job.configuration.get('options', [])
+
+    # Parse the custom options if there are any
+    for option in dracut_options:
+        # Deduplication check
+        if option not in cli_options:
+            cli_options.append(option)
+
+    if initramfs_name:
+        cli_options.append(initramfs_name)
```

Comment:
If we give people the ability specify options, I don't think we should force `-f`.  We should make it the default, but allow them to remove it if they choose to do so.

This is just my thought on how it could look but feel free to write in your own style if you prefer something different.

```suggestion
    # Fetch the job configuration
    initramfs_name = libcalamares.job.configuration.get('initramfsName', None)
    dracut_options = libcalamares.job.configuration.get('options', ["-f"])
    
    if initramfs_name:
        dracut_options.append(initramfs_name)
```